src/collections/client.ts

```diff
@@ -67,7 +65,8 @@ export class Client {
         applicationToken: parsedUri.applicationToken,
         baseApiPath: parsedUri.baseApiPath,
         keyspaceName: parsedUri.keyspaceName,
-        logLevel: parsedUri.logLevel
+        logLevel: parsedUri.logLevel,
+        authHeaderName: parsedUri.authHeaderName,
```

Comment:
Would be great to add an `options` parameter to `connect()`, so you can set `logLevel` and `authHeaderName` using a JavaScript object rather than relying on connection string.

Reply:
Makes sense, since the list keeps growing. And also, we have to pass options so we can pass username and password as well, as per another issue we created.

How about having `applicationToken`, `baseApiPath` and `keyspaceName` also as part of the options? 
If it is found in options, let's use that, otherwise let's use the values parsed from URL for `applicationToken`, `baseApiPath` and `keyspaceName` and regarding `logLevel` and `authHeaderName` we can just leave them in the options.

I think, this will be in sync with the logic of providing username password as part of the url or providing it as part of the options to override the url values.